### PR TITLE
fix(TBD-15244):'Check Services' button in metadata should not be visi…

### PR DESCRIPTION
**Pull request by dicarcab**
…ble in Synapse

**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/hadoopcluster/ui/StandardHCInfoForm.java [hunk 1]
```diff
@@ -2788,6 +2788,11 @@ private void hideFieldsOnSparkMode() {
             visibleGroupsBySparkMode.put(ESparkMode.SYNAPSE, Arrays.asList(synapseGroup));
             visibleGroupsBySparkMode.put(ESparkMode.HDI, Arrays.asList(hdiGroup, livyGroup, fsGroup));
 
+            if(sparkModeLabelName.toLowerCase().contentEquals("synapse")){
+                checkServicesBtn.setVisible(false);
+            }else{
+                checkServicesBtn.setVisible(true);
+            }
```
Comment (lbourgeois) [inline main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/hadoopcluster/ui/StandardHCInfoForm.java:2795]:
```suggestion                checkServicesBtn.setVisible(!"synapse".contentEquals(sparkModeLabelName.toLowerCase()));
```
And I would move it at the end of the block to keep groups related code together.

File: main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/hadoopcluster/ui/StandardHCInfoForm.java [hunk 2]
```diff
@@ -2788,6 +2788,11 @@ private void hideFieldsOnSparkMode() {
             visibleGroupsBySparkMode.put(ESparkMode.SYNAPSE, Arrays.asList(synapseGroup));
             visibleGroupsBySparkMode.put(ESparkMode.HDI, Arrays.asList(hdiGroup, livyGroup, fsGroup));
 
+            if(sparkModeLabelName.toLowerCase().contentEquals("synapse")){
+                checkServicesBtn.setVisible(false);
```
Comment (sponomarova) [inline main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/hadoopcluster/ui/StandardHCInfoForm.java:2792]: Wondering why it's only disabled for synapse. 
As far as I understand "check services" button should be only enabled for "yarn cluster" mode in universal and disabled for all the rest. Can it be corrected? 

Maybe I did bad description in this jira. Sorry.

**Review by sponomarova — CHANGES_REQUESTED** [review]

File: main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/hadoopcluster/ui/StandardHCInfoForm.java (hunk 2 above)
Comment (dicarcab) [reply main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/hadoopcluster/ui/StandardHCInfoForm.java:2792]: Ok I will update this but just to be clear the idea is that the check service button be only visible in yarn cluster for universal ?

File: main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/hadoopcluster/ui/StandardHCInfoForm.java (hunk 1 above)
Comment (dicarcab) [reply main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/hadoopcluster/ui/StandardHCInfoForm.java:2795]: will do it. And yes dont know why I write the if condition like i did

File: main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/hadoopcluster/ui/StandardHCInfoForm.java (hunk 2 above)
Comment (sponomarova) [reply main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/hadoopcluster/ui/StandardHCInfoForm.java:2792]: yes, that's the idea. This option doesn't make a lot of sense for other spark modes like dataproc or HDI or CDE, etc

**Review by sponomarova — APPROVED** [review]

**Pull request merged**